content/influxdb/cloud/influxdb-templates/monitor-enterprise.md

```diff
@@ -0,0 +1,184 @@
+---
```

Comment:
Is this a duplicate of the document above: https://github.com/influxdata/docs-v2/pull/3078/files#diff-880efcbec26052e4ff2a01746f277ef0dbdebbf904b87c46054051715f094d16?

We'll likely just want to link readers to that guide.

Reply:
I also believe that I adjusted this document so that they're separate!